src/api/getMovie.js

```diff
@@ -0,0 +1,12 @@
+import apiKey from "./apiKey";
+
+const getMovie = async (id, signal, plot = "short") => {
```

Comment:
It is unclear what else you can add to the "plot"
I would create an enum with valid choices

```
export const PLOT_TYPES = {
  SHORT: 'short',
  // .. other choices
}

// Usage
 import { PLOT_TYPES } from './api';

getMovie({ plot: PLOT_TYPES.SHORT }
```